package.json

```diff
@@ -109,6 +109,7 @@
     "@commitlint/config-conventional": "^8.3.4",
```

Comment:
@ifiri why tests never pass?

Reply:
we should change `connect` code for that
because we will move tests into auth, I didn't add required fragments